src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssObject = {};
+
+  const noSpacesArray = sourceString.split(';');
```

Comment:
The variable name 'noSpacesArray' is misleading because the array elements may still contain spaces; they are only split by semicolons. Consider renaming it to 'declarationsArray' or similar to better reflect its contents.